common/config_test.go

```diff
@@ -0,0 +1,94 @@
+package common
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfig(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
+
+	assert.Nil(nil)
```

Comment:
oh lol. Yeah, I put  that there so that the checker would stop complaining about the local `assert` being unused while I was still writing the test. Thanks for catching this.